custom_components/alexa_media/alarm_control_panel.py

```diff
@@ -125,147 +125,30 @@ async def async_unload_entry(hass, entry) -> bool:
-            _LOGGER.debug("%s: No Alexa Guard entity found", self.account)
-
-    async def async_added_to_hass(self):
-        """Store register state change callback."""
-        try:
-            if not self.enabled:
-                return
-        except AttributeError:
-            pass
-        # Register event handler on bus
-        self._listener = async_dispatcher_connect(
-            self.hass,
-            f"{ALEXA_DOMAIN}_{hide_email(self._login.email)}"[0:32],
-            self._handle_event,
-        )
-        await self.async_update()
-
-    async def async_will_remove_from_hass(self):
-        """Prepare to remove entity."""
-        # Register event handler on bus
-        self._listener()
-
-    def _handle_event(self, event):
-        """Handle websocket events.
-
-        Used instead of polling.
-        """
-        try:
-            if not self.enabled:
-                return
-        except AttributeError:
-            pass
-        if "push_activity" in event:
-            async_call_later(
-                self.hass,
-                2,
-                lambda _: self.hass.async_create_task(
-                    self.async_update(no_throttle=True)
-                ),
-            )
-
-    @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
-    @_catch_login_errors
-    async def async_update(self):
-        """Update Guard state."""
-        try:
-            if not self.enabled:
-                return
-        except AttributeError:
-            pass
-        import json
-
-        if self._login.session.closed:
-            self._available = False
-            self._assumed_state = True
-            return
-        _LOGGER.debug("%s: Refreshing %s", self.account, self.name)
-        state = None
-        state_json = await self.alexa_api.get_guard_state(
-            self._login, self._appliance_id
-        )
-        # _LOGGER.debug("%s: state_json %s", self.account, state_json)
-        if state_json and "deviceStates" in state_json and state_json["deviceStates"]:
-            cap = state_json["deviceStates"][0]["capabilityStates"]
-            # _LOGGER.debug("%s: cap %s", self.account, cap)
-            for item_json in cap:
-                item = json.loads(item_json)
-                # _LOGGER.debug("%s: item %s", self.account, item)
-                if item["name"] == "armState":
-                    state = item["value"]
-                    # _LOGGER.debug("%s: state %s", self.account, state)
-        elif state_json["errors"]:
-            _LOGGER.debug(
-                "%s: Error refreshing alarm_control_panel %s: %s",
+        self._attrs: Dict[Text, Text] = {}
+        _LOGGER.debug(
+                "%s: Guard Discovered %s: %s %s",
                 self.account,
-                self.name,
-                json.dumps(state_json["errors"]) if state_json else None,
+                self._friendly_name,
+                self._appliance_id,
+                self._guard_entity_id,
```

Comment:
For amp, we actually try to obfuscate unique identifiers if we can. At alexapy we can leave it raw. There should be an obfuscate function that will work here. I believe we left the last 5 digits.